content/departments/engineering/design/design_process.md

```diff
@@ -200,6 +200,8 @@ Use [Project Tools](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/Project-To
 
 - **Project Introduction** - necessary when sharing your prototype with the Team. It should contain details about the target audience and project goals, to provide good context before starting the review.
 
+- **Changelog** - it is used to track key design decisions and the reasoning behind them throughout the project. It is a source of truth for how the project evolved. Changelog keeps all the information in one place, so it's easier to find them during the project and when the team is working on the next iteration.
```

Comment:
```suggestion
- **Changelog** - it is used to track key design decisions and the reasoning behind them throughout the project. It is a source of truth for how the project evolved. Changelog keeps all the information in one place, so it's easier for others to review and understand the context for a projects design, and for when the team works on the next iteration.
```

Reply:
Thank you @jjinnii - I've added the 'review and understand' part :)